test/test_mypy.py

```diff
@@ -18,13 +18,25 @@
 import os
 import tempfile
 import unittest
-from typing import TYPE_CHECKING, Any, Dict, Iterable, Iterator, List
+from typing import TYPE_CHECKING, Any, Dict, Iterable, Iterator, List, Optional
 
 try:
-    from typing import TypedDict  # type: ignore[attr-defined]
+    from typing_extensions import NotRequired, TypedDict
```

Comment:
Doesn't this mean these tests will be skipped even when TypedDict exists because typing_extensions is not installed in CI?